components/coachindicator/stories/coachindicator.stories.js

```diff
@@ -82,3 +82,55 @@ export const Default = (args) => html`
 Default.args = {
 	variant: "default"
 };
+
+const DefaultVariantsGroup = (args) => html`
+	<div style="display: flex; flex-direction: column; padding: 16px">
+		${Template({
+				...args,
+				variant: "default"
+			})}
+		${Template({
+				...args,
+				variant: "dark"
+			})}
+		${Template({
+				...args,
+				variant: "light"
+				})}
+	</div>
+`;
```

Comment:
This is a similar approach to the checkbox. If there's a better way, let me know!

Reply:
see the refactor here: https://github.com/adobe/spectrum-css/pull/2793/commits/c64fc57964e1257d6ded8e5636f934fd7a2f829b